docs/reference/commandline/cp.md

```diff
@@ -23,7 +23,7 @@ You can copy from the container's file system to the local machine or the
 reverse, from the local filesystem to the container. If `-` is specified for
 either the `SRC_PATH` or `DEST_PATH`, you can also stream a tar archive from
 `STDIN` or to `STDOUT`. The `CONTAINER` can be a running or stopped container.
-The `SRC_PATH` or `DEST_PATH` be a file or directory.
+The `SRC_PATH` or `DEST_PATH` can be a file or directory.
```

Comment:
This whole paragraph switches voice halfway through. It goes from `is` to `can be`. The suggested change is an improvement, but the paragraph needs to be re-voiced as active.